x-pack/metricbeat/module/gcp/_meta/config.yml

```
@@ -60,3 +60,11 @@
   dataset_id: "dataset id"
   table_pattern: "table pattern"
   cost_type: "regular"
+- module: gcp
```

Nit: add a newline before this line for consistency with the rest of the file.
```suggestion

- module: gcp
```